src/EnergyPlus/SingleDuct.hh

```diff
@@ -403,20 +403,6 @@ struct SingleDuctData : BaseGlobalStruct
     Real64 ZoneTempSCV = 0.0;                      // Zone temperature [C]
     Real64 QMax2SCV = 0.0;
     int SysNumSATM = 0;
-    Real64 PriMassFlowRateCATM = 0.0;
-    Real64 PriEnthalpyCATM = 0.0;
-    Real64 PriHumRatCATM = 0.0;
-    Real64 PriTempCATM = 0.0;
-
-    Real64 SecAirMassFlowRateCATM = 0.0;
-    Real64 SecAirEnthalpyCATM = 0.0;
-    Real64 SecAirHumRatCATM = 0.0;
-    Real64 SecAirTempCATM = 0.0;
-
-    Real64 MixedAirMassFlowRateCATM = 0.0;
-    Real64 MixedAirEnthalpyCATM = 0.0;
-    Real64 MixedAirHumRatCATM = 0.0;
-    Real64 MixedAirTempCATM = 0.0;
```

Comment:
Eliminate state variables used only in function CalcATMixer.

Reply:
This is very very very nice.  Thanks.